### DISCUSSION TITLE
New App Icon Design

### DISCUSSION THREAD

**Discussion by LinuxGuy312** [z-huang/music#505] [Show and tell]
I've made a (maybe nice) temporary app icon in PixelLab :D
<img width=200 src="https://user-images.githubusercontent.com/101087324/211144505-6d67daf4-fc10-4e89-a67a-9b7b179e2418.png" />

**Comment by owenconnorz** [comment]
That's pretty good to be fair 👀

**Reply by LinuxGuy312** [reply]
Thanks 😄

**Comment by z-huang** [comment]
The design looks good, but it looks like "IJ".

**Reply by LinuxGuy312** [reply]
Well, as far as I know, music symbols don't really have anything that resembles "T" so this is the best I could do from my side 👀

**Comment by WirelessAlien** [comment]


**Reply by z-huang** [reply]
Thanks for your try, but I'm not in favor of it.

**Reply by WirelessAlien** [reply]
> Thanks for your try, but I'm not in favor of it.

Welcome. No problem.

**Comment by WirelessAlien** [comment]


**Reply by EisregenHaha** [reply]
Looks nice but I'm not a fan of this one really, looks a bit tacky while i think the logo should go more in the minalistic type
